samples/implicitQRAM/Program.qs

```diff
@@ -41,12 +42,12 @@
 //        }
 //    }
 
-    //Generates binary representation of [5,4,1]
-    function GenerateMemoryData() : Bool[][] {
-        let five = [true,false,true];
-        let four = [false,false,true];
-        let one = [false,false,true];
-        return [five, four, one];
+    //Generates binary representation of [5,4,1]=>[3,2,1]
+    function GenerateMemoryData() : (Bool[],Bool[])[] {
+        let fiveGivesThree = (IntAsBoolArray(5,3),IntAsBoolArray(3,2));
```

Comment:
If I understand the naming here correctly, this means that at address number 5, the value 3 is stored?  You used `Has` later in the tests and that might be more consistent?